compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -235,6 +235,9 @@ class ReplDriver(settings: Array[String],
             val newImports = newState.imports ++ extractImports(parsed.trees)
             val newStateWithImports = newState.copy(imports = newImports)
 
+            implicit val ctx: Context = newState.run.runContext
+            displayErrors(newState.run.runContext.flushBufferedMessages())
```

Comment:
Can you add a comment that this line display warnings. Otherwise it confusing (the name of the function is misleading).

Reply:
Can that function get renamed a bit?

Reply:
Sure, it's for another PR :)